docs/p12-limitation.rst

```diff
@@ -3,5 +3,21 @@
 Configure terminated HTTPS load balancer with PKCS12 certificate bundle
 =======================================================================
 
-Overview
---------
+F5 Neutron Lbaas Dashboard does not support load balancer SSL offloading function with PKCS 12 format certificates.
+
+The PKCS 12 bundle must be extracted before using F5 Neutron Lbaas Dashboard.
+
+- To extract a PKCS 12 bundle use openssl tool.
+
+.. code-block:: bash
+
+ # extract certificate with encrypt private keys
+ openssl pkcs12 -in certificate.pfx -out certificate.txt
+
+ # extract certificate without encrypt private key
+ openssl pkcs12 -in certificate.pfx -out certificate.txt -nodes
+
+- Make sure the extracted certificates are X509 format.
+- Copy corresponding certificates and private key from extracted file to F5 Neutron Lbaas Dashboard.
+- If the PKCS 12 bundle is extracted with encrypt private key, the new passphrase also needs to be provide to F5 Neutron Lbaas Dashboard.
+- Click ``create`` or ``update`` on the Dashboard to create or update SSL offloading for a load balancer.
```

Comment:
So we should make it clear where's the ``create`` or ``update``,  
SSL offloading for a load balancer.. for a listener??